examples/nextjs-trpc/src/env.js

```diff
@@ -0,0 +1,42 @@
+import { createEnv } from '@t3-oss/env-nextjs';
+import { z } from 'zod';
+
+export const env = createEnv({
+	/**
+	 * Specify your server-side environment variables schema here. This way you can ensure the app
+	 * isn't built with invalid env vars.
+	 */
+	server: {
+		NODE_ENV: z.enum(['development', 'test', 'production']),
+		ARCJET_KEY: z.string().startsWith('ajkey_').optional(),
```

Comment:
```suggestion
		ARCJET_KEY: z.string().startsWith('ajkey_'),
```
This doesn't need to be optional if you set the `SKIP_ENV_VALIDATION` environment variable in CI.